src/electionguard_cli/e2e_steps/e2e_step_base.py

```diff
@@ -8,10 +8,13 @@ class E2eStepBase:
     from the CLI.
     """
 
+    header_color = "green"
+    value_color = "yellow"
+
     def print_header(self, s: str) -> None:
```

Comment:
These functions to keep them in this file I added the header and value color as settable in the class. Truthfully, this should not be a base class like this. This should be a module with print functions. I will likely make an issue for this.